packages/sdk/src/sdk/api/tracks/types.ts

```diff
@@ -213,15 +243,15 @@ export const createUpdateTrackSchema = () =>
       metadata: createUploadTrackMetadataSchema().strict().partial(),
       transcodePreview: z.optional(z.boolean()),
       coverArtFile: z.optional(ImageFile),
-      onProgress: z.optional(z.function().args(z.number()))
+      onProgress: z.optional(z.function())
```

Comment:
nit but we could get more strict? https://github.com/colinhacks/zod?tab=readme-ov-file#functions

Reply:
The reason we like strict zod schemas is because we want to avoid bunk data being indexed from sdk consumers who aren't using typescript. The function schemas don't provide much value because the function isn't data being indexed. Also I realized that the zod function schemas only check the values being passed to and returned from the function at runtime, so it would really only be validating our own code

Reply:
Yeah makes sense, I was mostly considering the auto types benefit